src/uu/expand/src/expand.rs

```diff
@@ -80,10 +86,9 @@ impl Options {
             .unwrap(); // length of tabstops is guaranteed >= 1
         let tspaces = repeat(' ').take(nspaces).collect();
 
-        let files = if matches.free.is_empty() {
-            vec!["-".to_owned()]
-        } else {
-            matches.free
+        let files: Vec<String> = match matches.values_of("files") {
+            Some(s) => s.map(|v| v.to_string()).collect(),
+            None => vec!["-".to_owned()],
```

Comment:
Would be great to have a test covering this case :)

Reply:
Add test in f608084